wrangler-core/src/test/java/io/cdap/directives/column/SetTypeTest.java

```diff
@@ -213,6 +213,17 @@ public void testToDecimalInvalidRoundingMode() throws Exception {
     TestingRig.execute(directives, rows);
   }
 
+  @Test
```

Comment:
This test should also check whether output schema is generated correctly with default scale and precision.

Reply:
Done